docs/articles/documentation/recipes/integrating-testcafe-with-ci-systems/gitlab.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: docs
+title: Integrate TestCafe with GitLab
+permalink: /documentation/recipes/integrating-testcafe-with-ci-systems/gitlab.html
+---
+# Integrate TestCafe with GitLab
+
+This topic describes how to integrate TestCafe tests into a [GitLab](https://gitlab.com) project's build process.
+
+GitLab offers a built-in continuous integration service. You can read more about it in [GitLab documentation](https://docs.gitlab.com/ee/ci/quick_start/README.html).
+
+To run tests with TestCafe, you need a virtual machine with Node.js, TestCafe and browsers. Instead of installing them manually, you can use a [Docker](https://www.docker.com/) image.
```

Comment:
You can use a [Docker](https://www.docker.com/) image instead of installing them.